atomic_client.py

```diff
@@ -77,6 +77,10 @@ def images(self):
     def vulnerable(self):
         return self.dbus_object.VulnerableInfo(dbus_interface="org.atomic", timeout = 2147400)
 
+    @polkit.enable_proxy
+    def ps(self):
+        return self.dbus_object.Ps(dbus_interface="org.atomic", timeout = 2147400)
```

Comment:
Yes of course, i understand that ... but that timeout seems arbitrary and therefore could also be problematic.  would martin's approach here make sense?